kernel/cClosure.ml

```diff
@@ -28,14 +28,11 @@ open Util
 open Names
 open Constr
 open Declarations
-open Context
 open Environ
 open Vars
 open Esubst
 open RedFlags
 
-module RelDecl = Context.Rel.Declaration
```

Comment:
Removing opens is debatable but not using this alias doesn't count as cleanup IMO

Reply:
If anything, this is not very consistent: should there be a `NamedDecl` alias too?

Reply:
Just undid that part, and added  `NameDecl`.